### DISCUSSION TITLE
Capacitors

### DISCUSSION THREAD

**Discussion by seattlebuster** [gpstar81/GPStar-proton-pack#43] [General]
Saw that capacitors are recommended now.

Have you experienced issues or failures not having them?  I finished my pack, wondering if I should open it back up for these additions?

Thoughts?

**Comment by gpstar81** [comment]
> Saw that capacitors are recommended now.
> 
> 
> Have you experienced issues or failures not having them?  I finished my pack, wondering if I should open it back up for these additions?
> 
> Thoughts?

I had no issues with my testing. It is just good practice to have them with addressable LED strips.

**Reply by seattlebuster** [reply]
Good idea! I'll add the capacitors on pack #2 (84)

**Reply by gpstar81** [reply]
You can read more about why here if your curious:

https://electronics.stackexchange.com/questions/374922/why-would-i-need-a-capacitor-in-front-of-a-led-strip

https://forum.arduino.cc/t/neopixels-with-or-without-capacitor/609672

In short, it doesn't hurt to add them and they are cheap and it's good practice.
